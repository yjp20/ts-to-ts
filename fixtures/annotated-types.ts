/** @model */
type ConvertMe = string;

type SkipMe = number;

/** @model */
interface UserInfo {
  name: string;
  age: number;
}

interface IgnoreMe {
  data: string;
}
