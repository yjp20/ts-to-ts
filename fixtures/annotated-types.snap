<<<<<<<<<<< typescript <<<<<<<<<<<<
/** @model */
type ConvertMe = string;

type SkipMe = number;

/** @model */
interface UserInfo {
  name: string;
  age: number;
}

interface IgnoreMe {
  data: string;
}
===================================
// annotated-types.ts
alias ConvertMe = string;

model UserInfo {
  name: ConvertMe;
  age: float64;
};
>>>>>>>>>>> typespec >>>>>>>>>>>>>>